tools/cross_compile.sh

```diff
@@ -75,6 +76,36 @@ if [ "${OS}" == "android" ]; then
 
     ninja -v
     ninja package
+elif [ "${OS}" == "macos" ]; then 
+		    [[ "${BUILD_TYPE}" = "Release" ]] && suffix="" || suffix="-dbg"
```

Comment:
Indentation is off

Reply:
Thank you. This is should not be there at all. Macos builds should be initiated in build-cocoa.sh